tests/ttnn/unit_tests/operations/test_moreh_softmin.py

```diff
@@ -397,3 +389,84 @@ def test_softmin_backward_optional_output_tensor(shape_dim, optional_output_tens
     passing, out = comp_allclose_and_pcc(x.grad, tt_dev, rtol=rtol, atol=atol)
     logger.info(out)
     assert passing
+
+
+@pytest.mark.parametrize(
+    "shape_dim",
+    [
+        [[32, 32], 1],  # H small
+        [[32, 32], 0],  # W small
+        [[2, 3, 32 * 4, 32 * 5], 3],  # H large
+        [[2, 3, 32 * 4, 32 * 5], 2],  # W large,
+        [[1, 15, 32, 32], 1],  # C
+    ],
+)
+def test_softmin_callback(shape_dim, device, use_program_cache):
```

Comment:
It doesn't seem guaranteed that H large and W large will be called with the given input sizes. As you did in the `softmax_callback` test, it would be better to pass the strategy as an input to ensure that a specific factory is called.

Reply:
Resolved with adding strategy tests